exercises/concept/port-palermo/.meta/config.json

```diff
@@ -1,6 +1,7 @@
 {
   "authors": [
-    "meatball133"
+    "meatball133",
+    "egemen-dev"
```

Comment:
This should probably be "not authors" but "contributors" instead.

Reply:
Yes, much better.